Gemfile.lock

```diff
@@ -473,4 +473,4 @@ DEPENDENCIES
   webpacker
 
 BUNDLED WITH
-   2.2.17
+   2.2.18
```

Comment:
is this change related to this feature?
if updating here, probably have to update in docker image too